### PR TITLE
feat: limitted payload size

### PR DESCRIPTION
**Pull request by jonaphael**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)

* **Other information**:

### CODE REVIEW

File: src/mosca.js [hunk 1]
```diff
@@ -143,6 +143,20 @@ class MqttBackend {
     return topic.split('/')[index];
   }
 
+  /**
+   * Check received payload size
+   * 
+   * @param {obj} data data received *must be a string
+   * @param {obj} size number of bytes to compare
```
Comment (giovannicuriel) [inline src/mosca.js]: You can set the type as Number.

File: src/config.js [hunk 2]
```diff
@@ -31,4 +31,7 @@ config.healthcheck = {
     }
 };
 
+config.deviceToDojotPayloadSize = process.env.DEV_TO_DOJOT_PAYLOAD_MAX_SIZE || 256000000;
```
Comment (muhamedcpqd) [inline src/config.js]: Maybe here you can comment that value is in bytes (just for clarify). This value (256 kB) is specified  or is just an application requisition?

Comment (jonaphael) [reply src/config.js]: Thanks for the suggestion, I will add the comment, This value is specified, and dojot consider 256Mb as default for this module. but the max payload size can run up to 268,435,456 bytes.

Comment (muhamedcpqd) [reply src/config.js]: ok

**Pull request merged**